back/engines/commercial/multi_tenancy/lib/tasks/core/create_tenant.rake

```diff
@@ -352,6 +352,10 @@ namespace :cl2_back do # rubocop:disable Metrics/BlockLength
         follow: {
           enabled: true,
           allowed: true
+        },
+        user_session_recording: {
+          enabled: false,
+          allowed: false
```

Comment:
Same here, but indeed good to keep enabled on false, or it will mess with e2e tests.